### PR TITLE
teach with marker

### PR DESCRIPTION
**Pull request by cpaxton**

### CODE REVIEW

File: costar_robot/costar_robot_manager/launch/simple_iiwa_driver.launch [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
   <arg name="model" default="iiwa14"/>
 
   <include file="$(find iiwa_description)/launch/$(arg model)_upload.launch"/>
 
   <!--<node pkg="tf" type="static_transform_publisher" name="world_to_base_link" args=" 0 0 0 0 0 0 /world /base_link 1000"/>-->
-  <node pkg="tf" type="static_transform_publisher" name="kuka_to_base_link" args=" 0 0 0 0 0 0 1 /iiwa_link_0 /base_link 1000"/>
+  
+<!-- 
+<node pkg="tf" type="static_transform_publisher" name="kuka_to_base_link" args=" 0 0 0 0 0 0 1 /iiwa_link_0 /base_link 1000"/>
   <node pkg="tf" type="static_transform_publisher" name="ee_link_to_endpoint" args=" 0 0 0 0 0 0 1 /iiwa_link_ee /endpoint 1000"/>
```
Comment (cpaxton) [inline costar_robot/costar_robot_manager/launch/simple_iiwa_driver.launch]: delete this code

File: costar_robot/costar_robot_manager/launch/simple_psm_driver.launch [hunk 2]
```diff
@@ -1,21 +1,11 @@
...
-
-  <!--for "name" and "args" I just modified according to my understanding of rosparam ../robot_description-->
-  <node pkg="tf" type="static_transform_publisher" name="outer_yaw_to_base_link" args=" 0 0 0 0 0 0 1 /PSM1_psm_base_link /base_link 1000"/>
-  <node pkg="tf" type="static_transform_publisher" name="shaft_link_to_tool_tip" args=" 0 0 0 0 0 0 1 /PSM1_tool_tip_link /endpoint 1000"/>
-
   <node pkg="costar_robot_manager" name="simple_psm_driver_node" type="psm_driver.py" output="screen">
-    <remap from="robot_description" to="/dvrk/PSM1/robot_description"/>
-  </node>
-  
-  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher">
```
Comment (cpaxton) [inline costar_robot/costar_robot_manager/launch/simple_psm_driver.launch]: delete old code

File: costar_robot/costar_robot_manager/src/costar_robot/costar_arm.py [hunk 3]
```diff
@@ -379,7 +379,8 @@ def servo_to_pose_call(self,req):
             T = pm.fromMsg(req.target)
 
             # Check acceleration and velocity limits
-            (acceleration, velocity) = self.check_req_speed_params(req) 
+            (acceleration, velocity) = self.check_req_speed_params(req)
+            print "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!", T
```
Comment (cpaxton) [inline costar_robot/costar_robot_manager/src/costar_robot/costar_arm.py]: comment out or delete

File: costar_robot/costar_robot_manager/src/costar_robot/costar_arm.py [hunk 4]
```diff
@@ -459,8 +457,12 @@ def shutdown_arm_call(self,req):
     '''
     robot-specific logic to update state every "tick"
     '''
+    # TODO: Modify this part
     def handle_tick(self):
```
Comment (cpaxton) [inline costar_robot/costar_robot_manager/src/costar_robot/costar_arm.py]: do we still need this TODO?

File: costar_robot/costar_robot_manager/src/costar_robot/ur_driver.py [hunk 5]
```diff
@@ -154,35 +154,4 @@ def send_cart(self,q,acceleration,velocity):
...
 
-    def handle_tick(self):
-
-        # send out the joint states
-        #self.current_joint_positions = self.ur.getj_all()
-        #self.q0 = np.array(self.current_joint_positions[0])
-        #self.q_v0 = np.array(self.current_joint_positions[1])
-        #self.q_a0 = np.array(self.current_joint_positions[2])
-        #self.js_publisher.publish(JointState(
-        #  header=Header(stamp=rospy.Time.now()),
```
Comment (cpaxton) [inline costar_robot/costar_robot_manager/src/costar_robot/ur_driver.py]: we don't need any of this right?

Comment (cpaxton) [inline costar_robot/costar_robot_manager/src/costar_robot/ur_driver.py]: just want to make sure you meant to delete it

File: costar_robot/costar_robot_manager/launch/simple_iiwa_driver.launch (hunk 1 above)
Comment (bcjiang) [inline costar_robot/costar_robot_manager/launch/simple_iiwa_driver.launch]: You mean delete the commented parts or the whole launch file?

File: costar_robot/costar_robot_manager/launch/simple_psm_driver.launch (hunk 2 above)
Comment (bcjiang) [inline costar_robot/costar_robot_manager/launch/simple_psm_driver.launch]: Delete the file?

File: costar_robot/costar_robot_manager/src/costar_robot/costar_arm.py (hunk 4 above)
Comment (bcjiang) [inline costar_robot/costar_robot_manager/src/costar_robot/costar_arm.py]: I think not, we can modify things in psm class

File: costar_robot/costar_robot_manager/src/costar_robot/ur_driver.py (hunk 5 above)
Comment (bcjiang) [inline costar_robot/costar_robot_manager/src/costar_robot/ur_driver.py]: Ah maybe it's a mistake, I shouldn't touch ur5 codes

File: costar_robot/costar_robot_manager/launch/simple_iiwa_driver.launch (hunk 1 above)
Comment (cpaxton) [inline costar_robot/costar_robot_manager/launch/simple_iiwa_driver.launch]: commented parts

File: costar_robot/costar_robot_manager/launch/simple_psm_driver.launch (hunk 2 above)
Comment (cpaxton) [inline costar_robot/costar_robot_manager/launch/simple_psm_driver.launch]: ignore this comment actually

File: costar_robot/costar_robot_manager/src/costar_robot/ur_driver.py (hunk 5 above)
Comment (cpaxton) [inline costar_robot/costar_robot_manager/src/costar_robot/ur_driver.py]: i mightve deleted it when we were working together though...  the deleted code does not seem to do anything.

**Pull request merged**